cmd/create/idp/cmd.go

```diff
@@ -258,13 +266,26 @@ func init() {
 		"OpenID: List of scopes to request, in addition to the 'openid' scope, during the authorization token request.\n",
 	)
 
+	// HTPasswd
+	flags.StringVar(
+		&args.htpasswdUsername,
+		"username",
+		"",
+		"HTPasswd: Username to log into the cluster console with.\n",
+	)
+	flags.StringVar(
+		&args.htpasswdPassword,
+		"password",
+		"",
+		"HTPasswd: Password for provided username, to log into the cluster console with.\n",
+	)
+
 	interactive.AddFlag(flags)
+	reporter = rprtr.CreateReporterOrExit()
+	logger = logging.CreateLoggerOrExit(reporter)
```

Comment:
I would move these back inside the `run` function to be consistent, unless there's a reason to move it here?